crime-assessment-service/src/main/java/uk/gov/justice/laa/crime/assessmentservice/iojappeal/controller/IojAppealController.java

```diff
@@ -55,13 +53,17 @@ public ResponseEntity<ApiGetIojAppealResponse> getAppealByLegacyAppealId(@PathVa
     }
 
     @PostMapping
-    @Operation(description = "Create a new IoJ Appeal")
-    @ApiResponse(responseCode = "501")
     public ResponseEntity<ApiCreateIojAppealResponse> create(@RequestBody ApiCreateIojAppealRequest request) {
         List<String> validationErrors = ApiCreateIojAppealRequestValidator.validateRequest(request);
         if (!validationErrors.isEmpty()) {
             throw new CrimeValidationException(validationErrors);
         }
-        return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+        // Call dual-write method.
+        var appealEntity = mixedTransactionService.createIojAppeal(request);
+
+        var response = new ApiCreateIojAppealResponse();
```

Comment:
You might be able to streamline this a bit with something like:

`new ApiCreateIojAppealResponse().withAppealId(etc).withLegacyAppealId(etc)... `